cluster/gce/gci/configure-helper.sh

```diff
@@ -2207,6 +2207,8 @@ function start-kube-controller-manager {
   fi
   if [[ -n "${RUN_CONTROLLERS:-}" ]]; then
     params+=("--controllers=${RUN_CONTROLLERS}")
+  elif [[ "${CLOUD_PROVIDER_FLAG:-external}" == "external" ]]; then
+    params+=("--controllers=*,-nodeipam")
```

Comment:
TODO: remove if we merge https://github.com/kubernetes/kubernetes/pull/114596

Reply:
this PR is WIP, I prefer to get the fixes in before moving it in tree

Reply:
should I remove this now, since the PR merged?